routes/web.php

```diff
@@ -13,6 +14,6 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
+Route::controller(CityController::class)->group(function () {
```

Comment:
No need for a group here. Groups are meant for sharing route attributes with multiple routes. In this case, tho, there's only one route, which means you can get rid of the group and just have the route definition.